### DISCUSSION TITLE
Question about Worldguard

### DISCUSSION THREAD

**Discussion by Valcoxic** [GriefPrevention/GriefPrevention#735] [Q&A] [answered]
Hello there,

I'm using your plugin and worldguard and I see on the documentation that default it is not possible to claim in a worldguard region. But people can still claim in worldguard regions but they cannot build there so that is good. But how do I stop people for claiming the land?

Gr. Valcoxic

**Comment by RoboMWM** [comment]
Use the proper flag, I think it's passthrough not build.

I'm not really supporting the worldguard feature (as it really should be in an addon) so I'm not sure how it works all that much.

**Comment by Jikoo** [comment, marked as answer]
Players must be able to build in a WG region (flag BUILD) to be able to claim it. If you're having trouble with the WG integration, please make sure that WG and GP are both up to date - around the time of the 7.0.0 release WG made several changes in their API that broke compatibility.
